### PR TITLE
added files for calculating weighted scores

### PR DESCRIPTION
**Pull request by A-Baji**

### CODE REVIEW

File: utils.py [hunk 1]
```diff
@@ -126,6 +126,20 @@ def get_config(config_key):
             'description': 'The whole test analysis with all results',
             'type': 'list of items'
         },
+        {
+            'id': 7,
+            'name': 'Weighted scores',
+            'short_name': 'weighted_s',
```
Comment (farrukhshahzad) [inline utils.py:132]: better to keep full name like `weighted_scores`

**Review by farrukhshahzad — CHANGES_REQUESTED** [review]

**Comment by farrukhshahzad** [comment]
Looks very good. Great to see following PE8!

**Comment by farrukhshahzad** [comment]
Should you also return these weighted scores in AnalyzeTest?

File: utils.py (hunk 1 above)
Comment (A-Baji) [reply utils.py:132]: Oh yes. I will update.

File: utils.py [hunk 2]
```diff
@@ -126,6 +126,27 @@ def get_config(config_key):
             'description': 'The whole test analysis with all results',
             'type': 'list of items'
         },
+        {
+            'id': 7,
+            'name': 'weighted_scores',
+            'short_name': 'weighted_s',
```
Comment (farrukhshahzad) [inline utils.py:132]: We use `short_name`. Please make this `weighted_scores`. `name` is service name and could be any text.

**Review by farrukhshahzad — APPROVED** [review]

**Pull request merged**